### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by dylanuys**
- Upgrading Bittensor to 8.4.1 
- Separating image/video performance metric logging

### CODE REVIEW

File: start_miner.sh [hunk 1]
```diff
@@ -22,4 +22,5 @@ pm2 start neurons/miner.py --name bitmind_miner -- \
   --wallet.name $WALLET_NAME \
   --wallet.hotkey $WALLET_HOTKEY \
   --axon.port $MINER_AXON_PORT \
-  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT
+  --blacklist.force_validator_permit $BLACKLIST_FORCE_VALIDATOR_PERMIT \
+  --logging.debug
```
Comment (kenobijon) [inline start_miner.sh]: do we want to run in debug mode by default?

File: bitmind/validator/reward.py [hunk 2]
```diff
@@ -4,69 +4,86 @@
...
-        challenge_modality: str,
-        performance_trackers,
-    ) -> np.array:
+    label: float,
+    responses: List[float],
+    uids: List[int],
+    axons: List[bt.axon],
+    challenge_modality: str,
+    performance_trackers: Dict[str, Any]
+) -> Tuple[np.ndarray, List[Dict[str, Dict[str, float]]]]:
```
Comment (kenobijon) [inline bitmind/validator/reward.py:47]: is this necessary to also return metrics in get_rewards?

**Review by kenobijon — COMMENTED** [review]
couple small q's but besides that looks good

File: start_miner.sh (hunk 1 above)
Comment (dylanuys) [reply start_miner.sh]: info is better actually, switching to that now

File: bitmind/validator/reward.py (hunk 2 above)
Comment (dylanuys) [reply bitmind/validator/reward.py:47]: Yep get_rewards has been the function responsible for using the miner_performance_tracker object to compute metrics

**Review by kenobijon — APPROVED** [review]

**Pull request merged**